api/auth.go

```
@@ -74,7 +74,7 @@ func (c *client) Authorize(ctx context.Context, callback func(error)) (*Authoriz
 	port := ""
 	for i := range ports {
 		port = strconv.Itoa(ports[i])
-		ln, err = net.Listen("tcp", ":"+port)
+		ln, err = net.Listen("tcp", "127.0.0.1:"+port)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)